setup-tdx-host.sh

```diff
@@ -90,7 +118,12 @@ rm -f /etc/apt/apt.conf.d/99unattended-upgrades-kobuk
 # stop at error
 set -e
 
-add_kobuk_ppa ${TDX_PPA:-tdx-release}
+if [ "${dev:-0}" -eq "1" ]; then
+  add_kobuk_ppa "tdx-proposed"
```

Comment:
@ThibF We already can tune the TDX_PPA by modifying the setup config file : https://github.com/canonical/tdx/blob/5e4a70877028bd73fa02b218eb5a502f96d05f6e/setup-tdx-config#L26

I m wondering if we have to add an argument to the script or take the advantage of the current configuration mechanism to achieve this need; 

We could make the MR to accept the `TDX_PPA` field to accept a list of PPAs

```
TDX_PPA=tdx tdx-proposed
```

Reply:
I would prefer both as command line help is a also natural way to discover script capabilities.
I added also a line about the configuration to make it more visible.
I updated my PR to support a list of PPA, but made it compatible that you can either use the flag or the config file.

Reply:
Sure; i would think that it is a good point to not add more required feature to the scripts; that allows to lower the maintenance cost and futhermore, the conf approach is more flexible. But ok, let me think about it

Reply:
If we want the script argument, we will need to add the capability that allow  `setup-tdx-guest.sh` being called in `dev` mode when we create an image, this is the principal use case of `setup-tdx-guest.sh`